game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt

```diff
@@ -52,7 +52,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_percent_add"                              "11"
+        "damage_percent_add"                              "12" //OAA
```

Comment:
Leave the percentage unchanged. Even a 1% increase in the damge of this ultimate is huge. You should increase the base damage of this spell.